web/files/files_test.go

```diff
@@ -508,6 +532,37 @@ func TestUploadWithDate(t *testing.T) {
 	assert.Equal(t, "2016-09-19T12:38:04Z", updatedAt)
 }
 
+func TestUploadWithCreatedAtAndUpdatedAt(t *testing.T) {
+	buf := strings.NewReader("foo")
+	req, err := http.NewRequest("POST", ts.URL+"/files/?Type=file&Name=TestUploadWithCreatedAtAndUpdatedAt&CreatedAt=2016-09-18T10:24:53Z&UpdatedAt=2020-05-12T12:25:00Z", buf)
+	req.Header.Add(echo.HeaderAuthorization, "Bearer "+token)
+	assert.NoError(t, err)
+	res, obj := doUploadOrMod(t, req, "text/plain", "rL0Y20zC+Fzt72VPzMSk2A==")
```

Comment:
Nit: What does `rL0Y20zC+Fzt72VPzMSk2A==` mean ?

Reply:
md5 binary hash of the "foo" string converted to base64.